includes/admin/payments/class-refund-items-table.php

```diff
@@ -191,59 +249,167 @@ private function format_currency( $order_item, $column_name ) {
+				<?php
+				if ( 'after' === $currency_pos ) {
+					echo '<span class="edd-amount-control__currency is-after">';
+					echo esc_html( $this->get_currency_symbol( $item->order_id ) );
+					echo '</span>';
+				}
+				?>
+			</div>
+			<small class="edd-order-item-refund-max-amount">
+				<?php
+				echo _x( 'Max:', 'Maximum input amount', 'easy-digital-downloads' ) . '&nbsp;';
+
+				echo $this->format_currency( $item, $column_name, $amount_remaining );
+				?>
+			</small>
+		</div>
+		<?php
+		return ob_get_clean();
+	}
+
+	/**
+	 * Retrieves the currency symbol for a given order item.
 	 *
-	 * @param \EDD\Orders\Order_Item $order_item Order Item object.
+	 * @param int $order_id
 	 *
-	 * @return string Data shown in the Name column
+	 * @since 3.0
+	 * @return string|null
 	 */
-	public function column_name( $order_item ) {
-		// Wrap order_item title in strong anchor
-		$order_item_title = '<strong>' . $order_item->get_order_item_name() . '</strong>';
-		if ( 'refunded' !== $order_item->status ) {
-			$order_item_title = '<label for="edd-order-item-' . esc_attr( $order_item->id ) . '" class="edd-order-item__label">' . $order_item->get_order_item_name() . '</label>';
+	private function get_currency_symbol( $order_id ) {
+		static $symbol = null;
+
+		if ( is_null( $symbol ) ) {
+			$order = edd_get_order( $order_id );
+
+			if ( $order ) {
+				$symbol = edd_currency_symbol( $order->currency );
+			}
 		}
 
-		return $order_item_title;
+		return $symbol;
 	}
 
 	/**
 	 * Render the checkbox column
 	 *
 	 * @since 3.0
 	 *
-	 * @param \EDD\Orders\Order_Item $order_item Order Item object.
+	 * @param Order_Item|Order_Adjustment $item Order Item or Order Adjustment object.
 	 *
-	 * @return string Displays a checkbox
+	 * @return string
 	 */
-	public function column_cb( $order_item ) {
-		if ( 'refunded' !== $order_item->status ) {
+	public function column_cb( $item ) {
+		$item_id            = $this->get_item_unique_id( $item );
+		$object_type        = $this->get_object_type( $item );
+		$item_quantity      = $item instanceof Order_Item ? $item->quantity : 1;
+		$refundable_amounts = $item->get_refundable_amounts();
+		$total_remaining    = array_key_exists( 'total', $refundable_amounts ) ? floatval( $refundable_amounts['total'] ) : 0.00;
+
+		if ( 'refunded' !== $item->status && $total_remaining > 0 ) {
```

Comment:
This logic is working with credits, discounts, and positive fees, but fails with negative fees. I'm trying this with Discounts Pro and it's failing because the `$total_remaining` is negative.